apps/builder/tsconfig.json

```diff
@@ -10,6 +10,7 @@
   "compilerOptions": {
     "paths": { "~/*": ["app/*"] },
     "baseUrl": ".",
-    "rootDir": "./app"
+    "rootDir": "./app",
+    "declaration": false
```

Comment:
Does this mean we stuck with rootDir option and cannot run checks on configurations in the root?

Reply:
No, I don't think so. I didn't change anything here - I simply want to disable declarations here. An app won't have declarations emitted for it so we don't care if types are portable here or not.